oas_apivideo.yaml

```diff
@@ -17458,6 +17458,110 @@ x-webhooks:
                   type: string
                   enum: [240p, 360p, 480p, 720p, 1080p]
                   example: 1080p
+      responses:
+        '202':
+          summary: Accepted
+          description: Your webhook server may return this response to api.video to signal that the webhook is accepted.
+  video.caption.generated:
+    post:
+      tags:
+        - Webhooks
+      summary: Video caption generated
+      description: This webhook triggers when the API finishes generating a caption for a video.
+      operationId: POST-webhooks
+      parameters:
+        - in: header
+          name: X-Api-Video-WebhookID
+          schema:
+            type: string
+          description: The unique ID of your webhook.
+          required: true
+        - in: header
+          name: X-Api-Video-Signature
+          schema:
+            type: string
+          description: The webhook's body encrypted using the webhook's signature secret, in HMAC SHA256. Use this hash to verify that api.video is the origin of this webhook notification.
+          required: true
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                type:
+                  type: string
+                  description: The name of the webhook event that occurred.
+                  example: video.caption.generated
+                emittedAt:
+                  description: Returns the date-time when the webhook event occurred.
+                  type: string
+                  format: date-time
+                  example: '2024-08-151T10:18:47+00:00'
+                videoId:
+                  description: The ID of the video where the caption was generated.
```

Comment:
I'd say "The ID of the video for which the caption was generated." I think!

Reply:
Fixed ✅